airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/base/AirbyteTraceMessageUtility.java

```diff
@@ -86,6 +92,14 @@ private static AirbyteMessage makeErrorTraceAirbyteMessage(
                 .withStackTrace(ExceptionUtils.getStackTrace(e))));
   }
 
+  private static AirbyteMessage makeAnalyticsTraceAirbyteMessage(final AirbyteAnalyticsTraceMessage airbyteAnalyticsTraceMessage) {
+    return new AirbyteMessage().withType(Type.TRACE)
```

Comment:
for my education:

1. if we send out a trace message, platform is supposed to send it to sentry?
2. if we want to throw out a specific error message, is that still done through this trace message?

Reply:
1. Platform forwards it to Segment, which uploads to our data warehouse. See https://docs.google.com/document/d/1ZYoCs79GAfTn-x0RL76r4gWm1IWXqJ0cMP5K1RUPeWM/edit#heading=h.fa72b39y2m99
2. Error messages are done via a different trace message. For this feature of CDC - it's true we eventually want to fail the sync. But first, we'd like to understand how many workspaces face this issue.